src/main/java/hlf/java/rest/client/service/impl/ChannelServiceImpl.java

```diff
@@ -371,11 +371,16 @@ private Configtx.ConfigGroup getMSPConfigGroup(hlf.java.rest.client.model.Peer p
         .build();
   }
 
-  // The method returns a default policy for each organization
-  // that maps the roles. The policy type is signature. Roles
-  // are identified by their signatures, as those signatures
-  // represent the certificate.
-  private HashMap<String, Configtx.ConfigPolicy> getDefaultRolePolicy(String orgMSPId) {
+  /**
+   * get default configuration policy for organization that maps the roles. The policy type is
+   * signature. Roles are identified by their signatures, as those signatures represent the
+   * certificate.
+   *
+   * @param orgMSPId Org MSP ID
+   * @return HashMap with role and the configuration policy
+   */
+  @Override
+  public HashMap<String, Configtx.ConfigPolicy> getDefaultRolePolicy(String orgMSPId) {
```

Comment:
[nitpick] What do you think?
Looks like this can be a channel utility function and it can be moved to a different class.

Reply:
Good suggestion, created a class for channel utility and moved the functions to this class so it can be used anywhere.